### ISSUE TITLE
Communicating channels: gitter.im + discourse

### ISSUE THREAD

**Issue by cauerego** [tensorflow/tensorflow#132]
If we're in for the openess and effectiveness of the whole thing...

(1) Why not using gitter.im ?

I also believe we should have a discourse forums, which I'm more than willing to help / set up if you really need (just can't pay for it or move people from the mailing list without being official).

(2) Yes, discourse would be a **perfect** replacement for the mailing list **and** to this Issue tracker. :)

**Comment by anshzinc** [comment]
Can you give example of a large OSS project that are using discourse effectively? thanks.

**Comment by cauerego** [comment]
@anshzinc other than discourse.org itself: http://discourse.ubuntu.com/

**Comment by bhack** [comment]
:+1:

**Comment by cauerego** [comment]
Should probably split this into 2 separate issues, shouldn't we?

**Comment by delip** [comment]
since the TF community is growing so rapidly, I'm not sure if the TF devs at Google will have bandwidth to scale with this growth. Just an idea: How about we start something unofficial where anyone interested in TF can log on and ask questions to the room? This will also cut down the number of duplicate issues we are seeing here.

**Comment by cauerego** [comment]
@delip I've tried doing such stuff before, a few times, including going with official routes, and I've always failed. I got a good grasp on how it should be done next to prevent my previous failures and I while I still think just *starting something unofficial* **might** work I would bet it won't. Because there is already this official channel that people already think "it's good enough".

This needs to be closed in favor of the better one. That's how it would surely work.
